src/clusterinfo.c

```diff
@@ -208,12 +208,17 @@ int loadbandmap(void) {
 
 	    g_strlcpy(spotcall, thisline + 26, 6);
 
-	    strncpy(spottime, thisline + 70, 4);	// how old?
-	    spottime[4] = spottime[3];
-	    spottime[3] = spottime[2];
-	    spottime[2] = ':';
-	    spottime[5] = '\0';
-	    spotminutes = 60 * atoi(spottime) + atoi(spottime + 3);
+	    /* read and convert hours and minutes to spotminutes */
+	    spottime[0] = *(thisline + 70);
```

Comment:
maybe `spottime[x] = thisline[y]` would be more readable

ideally one would need to check if `thisline` has enough content, something like strlen >= 74, on line 207 in the if()

Reply:
Right. Good. Idea. Wrt the length I will look into it.